tests/elements-regression/src/controls/media.js

```diff
@@ -0,0 +1,70 @@
+const { ControlBase } = require( './control-base' );
+
+class Media extends ControlBase {
+	getSelector() {
+		return `.elementor-control-${ this.config.name }.elementor-control-type-media`;
+	}
+
+	async getValue() {
+		await this.openMediaLibrary();
+
+		const selectedMediaId = await this.getSelectedMediaId();
+
+		const value = selectedMediaId
+			? Object.entries( process.env )
+				.find( ( [ key, id ] ) =>
+					key.startsWith( 'ELEMENTS_REGRESSION_MEDIA_IDS_' ) &&
+					id === selectedMediaId,
+				)
+				?.[ 0 ]
+				?.replace( 'ELEMENTS_REGRESSION_MEDIA_IDS_', '' )
+			: null;
+
+		await this.closeMediaLibrary();
+
+		return value;
+	}
+
+	async setValue( newValue ) {
+		const id = process.env?.[ `ELEMENTS_REGRESSION_MEDIA_IDS_${ newValue }` ];
```

Comment:
do you store the file name as the key? I don't think it's the best idea

Reply:
Done